.build/vendor-build.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
nit, filename: vendor_build.sh, or build_vendor.sh to conform with .build file naming?